client/js/components/NavigationDrawer.vue

```diff
@@ -65,7 +66,7 @@ export default {
             ...rules,
           ]
         : [
-            { text: 'Logout', icon: 'logout', page: { name: 'Login' } },
+            { text: 'Logout', icon: 'logout', page: { name: 'Logout' } },
```

Comment:
This was needed to ensure the user actually logs out. All this does it handle it via a middleware.